src/pages/workspace/WorkspaceBankAccountPage.js

```diff
@@ -19,12 +19,14 @@ import Section from '../../components/Section';
 import WorkspaceResetBankAccountModal from './WorkspaceResetBankAccountModal';
 import styles from '../../styles/styles';
 import CONST from '../../CONST';
+import withFullPolicy, {fullPolicyPropTypes} from './withFullPolicy';
 
 const propTypes = {
     /** ACH data for the withdrawal account actively being set up */
     reimbursementAccount: reimbursementAccountPropTypes,
 
     ...withLocalizePropTypes,
+    ...fullPolicyPropTypes,
```

Comment:
@mollfpr we don't need all the props.
So something like this would suffice
```
    /** Policy values needed in the component */
    policy: PropTypes.shape({
            name: PropTypes.string,
    }).isRequired,
```